casepro/contacts/tests.py

```diff
@@ -175,16 +198,33 @@ def test_as_json(self):
         self.assertEqual(self.ann.as_json(full=True), {
             'id': self.ann.pk,
             'name': "Ann",
+            'urns': [],
             'language': None,
             'groups': [{'id': self.reporters.pk, 'name': "Reporters"}],
             'fields': {'nickname': None, 'age': "32"},
             'blocked': False,
             'stopped': False
         })
 
-        # if site uses anon contacts then name is obscured
+        # If the contact has urns and the contact display uses urns then they should be returned and the name should be
+        # the first urn
+        self.ann.urns = ['tel:+234567890', 'mailto:ann@test.com']
+        self.ann.save()
+        with override_settings(SITE_CONTACT_DISPLAY='urn'):
+            self.assertEqual(self.ann.as_json(full=True), {
+                'id': self.ann.pk,
+                'name': "+234567890",
+                'urns': [{'scheme': "Phone", 'path': "+234567890"}, {'scheme': "Email", 'path': "ann@test.com"}],
```

Comment:
I think better to just return full URN strings here (i.e. "tel:+234567890"). We can split them in the frontend code and just use a hardcoded map to look up scheme names from scheme prefixes.

Otherwise if we made a contact editing form we'd need to translate these scheme names back into scheme prefixes. Also when a user submits a Forward to URN, we treat the URNs as full strings so good if this is consistent with that.